lib/src/polar_base.dart

```
@@ -125,6 +126,18 @@ class Polar {
           ),
         );
         return;
+      case 'offlineRecordingTriggerSet':
+        try {
+          await setOfflineRecordingTrigger(
+            call.arguments[0],
+          );
+        } catch (e) {
+          developer.log(
```

We should throw an exception here instead